### ISSUE TITLE
Plazi figures images etc. mediated data only discoverable through clicking on binomial url 

### ISSUE THREAD

**Issue by olafbanki** [CatalogueOfLife/checklistbank#1115]

**Comment by mdoering** [comment]
Create a new dataset search issue #1116

**Comment by olafbanki** [comment]
Yes I also realised later on it is not so much a portal issue. I agree with your suggestion that we need to rethink the dataset search. I understand that it is difficult showing an image next to a dataset in a search. However, maybe we can find a way to show at a high level that a dataset contains certain attributes (descriptions/treatments, images). It is the 1st high level discovery that I miss.
